README.md

```diff
@@ -56,9 +56,9 @@ defmodule MyApp.Emails do
 
   def welcome_email do
     base_email
-    # Bulk update the email
-    |> struct(bcc: "someone@bar.com", from: "other_person@foo.com")
-    |> to("foo@bar.com", ["John Smith": "john@foo.com"])
+    # A way to bulk update email attributes. Still figuring out
+    |> update_attrs(bcc: "someone@bar.com", from: "other_person@foo.com")
```

Comment:
It's not really metadata though. I would be ok with `update_email` even. This isn't implemented in code yet though because I'm not sure about it. It might even make sense to not have it and instead just recommend using `struct` to update it